draft-ietf-masque-connect-udp-listen.md

```diff
@@ -173,6 +298,22 @@ client. The proxy uses the same port to listen for UDP packets from any
 authorized target and encapsulates the packets in the Binding Datagram
 Payload format, specifying the IP and port of the target and forwards it to
 the client.
+When the client or proxy send a COMPRESSION_ASSIGN capsule, the proxy or client
+respectively either register a mapping from Context ID to the provided target
```

Comment:
This isn't quite right, the action on the receiver isn't when the sender sends, it's on reception

```suggestion
When an endpoint receives a COMPRESSION_ASSIGN capsule, it
either registers a mapping from Context ID to the provided target
```